src/alexapi/device_platforms/dummy.py

```diff
@@ -11,36 +14,28 @@ def __init__(self, config):
 		self.should_confirm_trigger = False
 
 	def setup(self):
-		if self._pconfig['verbose']:
```

Comment:
The reason I removed it from dummy is that no other platform module had such option, but they should. It is actually better to move that logging to `baseplatform`, so it can be enabled for any kind of platform.